charts/images.yaml

```diff
@@ -0,0 +1,8 @@
+images:
+  - name: runtime-gvisor
+    sourceRepository: github.com/gardener/gardener-extension-runtime-gvisor
+    repository: eu.gcr.io/gardener-project/gardener/extensions/runtime-gvisor
+  - name: runtime-gvisor-installation
+    sourceRepository: github.com/gardener/gardener-extension-runtime-gvisor
+    repository: eu.gcr.io/gardener-project/gardener/extensions/runtime-gvisor-installation
+    tag: v0.1.0-dev
```

Comment:
hm, remove it - it should be found automatically

Reply:
The installation image is used by the deamonSet deployed by the extension. 
It is comparable to [here in the provider extension](https://github.com/gardener/gardener-extension-provider-aws/blob/6d2236cb72744e148682a3454ba2938b1af92e37/charts/images.yaml#L2) - where the tags are also specified.

Reply:
The tag shall be removed and is found out dynamically in your function in `pkg/imagevector.go`.

Reply:
I see that that is neat for the local dev case so that the image name becomes something like  v0.1.0-dev-<commitID>. 
However when I deploy this extension with a controllerRegistration without specifying an image vector override - the version installation image version is in this case 0.0.0-dev and does not exist. 
It would always require to specify an image-vector override.
Is that how it is intended? 
Also I oriented myself on the provider-extension (see above) - why is the tag specified there?

Reply:
No, you have to make sure that the correct version is injected during the `go install/build` using the LD flags. 

> Also I oriented myself on the provider-extension (see above) - why is the tag specified there?

This is something completely different. The gvisor extension is the first extension that deploys an image that is built from its own repository. All other extensions deploy images built from other repositories.

Reply:
ok got it, thanks! 
Updated the PR accordingly. 
Please also see: [gardener PR](https://github.com/gardener/gardener/pull/2165)
- I will remove the replace command in go.mod as soon as the gardener PR is in.